lib/events/complete.go

```diff
@@ -281,6 +281,7 @@ loop:
 				desktopSessionEnd.DesktopName = fmt.Sprintf("%v (recovered)", e.DesktopName)
 
 			case *events.SessionStart:
+				sshSessionEnd.Metadata = e.Metadata
```

Comment:
His change seems to be not related to this PR but does the metadata are also not set for `events.WindowsDesktopSessionStart` ?

Reply:
This was not related to this PR. It was a debug code that I forgot